README.md

```diff
@@ -133,9 +136,21 @@ docker run -d \
   -v ./storage:/storage \
   -e HOSTD_SEED="my wallet seed" \
   -e HOSTD_API_PASSWORD=hostsarecool \
-    ghcr.io/siafoundation/hostd:latest
```

Comment:
`ghcr.io/siafoundation/hostd` is required to pull the correct image. We're not using Docker's container registry

Reply:
The reason I changed this line is because if a user builds the container using the docker file they shouldn't run with `ghcr.io/siafoundation/hostd` since they likely intended running their own copy. I included verbiage on line 124 to reference that image in case there is a user who does not want to build the container.